opportunity.py

```diff
@@ -0,0 +1,92 @@
+from dataclasses import dataclass
+from dotenv import load_dotenv
+import psycopg2
+from typing import List
+import os
+import utility as utils 
+
+load_dotenv()
+
+table_name = os.getenv("DB_TABLE")
+
+@dataclass
+class Opportunity:
+    """Class for ..."""
+    # TODO
```

Comment:
Here is an example `Opportunity` dataclass.
```suggestion
class Opportunity:
    _company: str
    _title: str
    _location: str
    _link: str
    _processed: bool
```